index.js

```diff
@@ -8,14 +8,24 @@ const clone = require('lodash.clonedeep');
 const uuid = require('uuid/v4');
 
 /**
+ * Proxy polyfill injection.
+ */
+
+require('proxy-polyfill');
```

Comment:
That is true, but the only way is to fork and export the Proxy object instead of mutating global namespace.